airflow/executors/celery_executor.py

```diff
@@ -41,6 +41,8 @@
 
 CELERY_SEND_ERR_MSG_HEADER = 'Error sending Celery task'
 
+SEND_TASK_TIMEOUT = conf.getint('celery', 'SEND_TASK_TIMEOUT')
```

Comment:
```suggestion
SEND_TASK_TIMEOUT = conf.getint('celery', 'send_task_timeout')
```
The key is case sensitive.

Reply:
really? i see all lower in cfg but lots of upper in python code

Reply:
I think since a recent release, the conf entries are no longer case sensitive. I see conf.get* with capitcal cases all over the place.

Reply:
<img width="720" alt="Screenshot 2020-01-13 at 12 03 35" src="https://user-images.githubusercontent.com/12058428/72250800-bf8de680-35fc-11ea-889c-10223c44636b.png">

Reply:
@mik-laj okay I see your point. I updated to all lower case.

However, I do see a lot of places where conf entries are queried like this:

e.g.
```
airflow/bin/cli.py
        'worker_timeout': Arg(
            ("-t", "--worker_timeout"),
            default=conf.get('webserver', 'WEB_SERVER_WORKER_TIMEOUT'),
            type=int,
            help="The timeout for waiting on webserver workers"),

airflow/utils/log/file_task_handler.py
            url = os.path.join(
                "http://{ti.hostname}:{worker_log_server_port}/log", log_relative_path
            ).format(
                ti=ti,
                worker_log_server_port=conf.get('celery', 'WORKER_LOG_SERVER_PORT')
            )

```

So something strange is going on. I did not spend much time looking into this thought since that's not what I originally intended to investigate. One possibility is this old change was not applied globally correctly. That's just my speculation: https://github.com/apache/airflow/pull/5668/files